app/views/reliefs/reliefsPrintFriendly.scala.html

```diff
@@ -234,53 +234,61 @@ <h1 class="heading-xlarge" id="property-details-summary-header">@messages("ated.
-      </td>
-    </tr>
-    <tr>
-      <td><p id="tas-er" class="heading-small">@messages("ated.relief-summary.avoidanceSchmRefNum")</p></td>
-      <td id="tas-er-val">@reliefData.reliefs.isAvoidanceScheme.fold("None")(x=>reliefData.taxAvoidance.equityReleaseScheme.fold("None")(a=>a))</td>
-    </tr>
-    <tr>
-      <td><p id="tas-promoter-er" class="heading-small">@messages("ated.relief-summary.avoidanceSchmRefNum-promoter")</p></td>
-      <td id="tas-promoter-er-val">@reliefData.taxAvoidance.equityReleaseSchemePromoter.fold("None")(x=>reliefData.taxAvoidance.equityReleaseSchemePromoter.fold("None")(a=>a))</td>
-    </tr>
-    <tr>
-      <td><p id="status-er" class="heading-small">@messages("ated.relief-summary.status")</p></td>
-      <td><p id="status-er-draft">@messages("ated.relief-summary.draft")</p></td>
-    </tr>
-    }
 
+      @if(reliefData.reliefs.equityRelease) {
+        <tr>
+          <td class="ated-table-padding-top"><p class="heading-medium" id="equityRelease">@messages("ated.relief-summary-types.equityRelease")</p></td>
+          <td class="ated-table-padding-top"></td>
+        </tr>
+        <tr>
+          <td><p id="taxAvd-dateofrel-er" class="heading-small">@messages("ated.relief-summary.datesOfRelief")</p></td>
+          <td id="dateFromTo-er">
+            @reliefData.reliefs.equityReleaseDate.getOrElse(reliefData.periodStartDate).toString(messages("ated.date-format.short"))  @messages("ated.relief-summary.relief-end-date")
+          </td>
+        </tr>
+        <tr>
+          <td><p id="tas-er" class="heading-small">@messages("ated.relief-summary.avoidanceSchmRefNum")</p></td>
+          <td id="tas-er-val">@reliefData.reliefs.isAvoidanceScheme.fold("None")(x=>reliefData.taxAvoidance.equityReleaseScheme.fold("None")(a=>a))</td>
+        </tr>
+        <tr>
+          <td><p id="tas-promoter-er" class="heading-small">@messages("ated.relief-summary.avoidanceSchmRefNum-promoter")</p></td>
+          <td id="tas-promoter-er-val">@reliefData.taxAvoidance.equityReleaseSchemePromoter.fold("None")(x=>reliefData.taxAvoidance.equityReleaseSchemePromoter.fold("None")(a=>a))</td>
+        </tr>
+        <tr>
+          <td><p id="status-er" class="heading-small">@messages("ated.relief-summary.status")</p></td>
+          <td><p id="status-er-draft">@messages("ated.relief-summary.draft")</p></td>
+        </tr>
+      }
     }
   </table>
 
   <p id="reliefs-print-charge-text">@messages("ated.relief-summary.table.ated-charge.text")</p>
 
-  <p id="reliefs-print-charge-value" class="heading-xlarge">@messages("ated.relief-summary.table.ated-charge.balance")</p>
+  <p id="reliefs-print-charge-value" class="heading-xlarge">
+  @if(isComplete) {
+    @messages("ated.relief-summary.table.ated-charge.balance")
+  } else {
+    @messages("ated.unknow-amount.text")}
```

Comment:
typo "unknown"

Reply:
I was utilising the existing message but have now refactored to unknown which has required a change to propertyDetailsSummary and reliefsSummary views
Retested locally and all tests still pass